prettier.config.mjs

```diff
@@ -0,0 +1,16 @@
+export default {
+  useTabs: false,
+  trailingComma: "es5",
+  tabWidth: 4,
+  semi: true,
+  singleQuote: false,
+  printWidth: 200,
```

Comment:
140 is already massive and IMHO too large, 200 is quite brutal. I would reduce it all to max 120. I'm raising it as prettier output is not clear at the moment anyway so reformat is needed.

Reply:
it's this project's default. I'm used to a lot less myself, I tried reducing it but it quickly becomes unreadable in vue templates. 

Overall prettier and eslint configuration could be improved a lot. Could be done outside of this PR